_protocols/beer-dna-sequencing.md

```diff
@@ -6,126 +6,161 @@ image: /images/protocols/beer-dna-sequencing.jpg
+## Sequencing
 
-#### Starting the software:
+1. Prepare the reagents
+    1. Mix the SQB tube by vortexing, spin down and return to ice
+    2. Mix the FB tube by vortexing, spin down and return to ice
+    3. Spin down the FLT tube, mix by pipetting, and return to ice
+2. Prepare loading port
+    1. Open priming port on the flowcell to check for small bubbles
+    2. Remove possible bubbles by taking 200 µl liquid from the port using the P1000 pipette
+    3. Remove the liquid by turning the wheel of the pipet but only until 220-230 µl
 
-Requirements:
-- You have to have a account at nanopor MinION to start the software
-- The divice need to be attached to the computer to start the software
+    Note: removing more than 30 µl will damage the pores in the array because they need to be covered by the buffer at all times
 
-Either you find the "MinION Software" botton directly or you open it via the terminal using the path "/opt/ui/MinKNOW" 
+3. Prepare the **priming mix**
+    1. Add 30 µl of Flush Tether (FLT) directly to the Flush Buffer (FB) Eppi tube
+    2. Mix by pipetting
 
-#### Launching MinKNOW and running the sequencing
+4. Load 800 μl of the priming mix into the flow cell via the priming port, avoiding introduction of air bubbles
+5. Wait for 5 minutes
 
-After having the software installed, the major steps would be connect, initialize and configure the MinION within the MinKNOW software. Afterwards the device would be ready for loading and sequencing the data.
+6. Prepare **library** for loading by mixing in a new tube
+    - 34 μl Sequencing Buffer (SQB)
+    - 25.5 μl Loading Beads (LB) (mix before adding)
+    - 4.5 μl nuclease-free water
+    - 11 μl DNA library
 
-- Again please refer to the community web site for up-to-date instructions and follow the graphically enhanced manuals.
-- This is the current link to the manual for starting MinKNOW and the follow up procedures. https://community.nanoporetech.com/protocols/experiment-companion-minknow/v/mke_1013_v1_revan_11apr2016/introduction-to-local-base
-- Since the URL is not generic, it would be best to find the latest manual from the getting started page here: https://community.nanoporetech.com/getting_started
+7. Load the library into the flow cell
+    1. Gently lift the SpotON sample port cover to make the SpotON sample port accessible
+    2. Load 200 μl of the priming mix into the flow cell via the **priming port (not the SpotON sample port)**, avoiding introduction of air bubbles
+    3. Mix the prepared library gently by pipetting up and down prior to loading
+    4. Add 75 μl of the library preparation to the flow cell via the SpotON sample port in a dropwise fashion
+    
+        Ensure each drop flows into the port before adding the next
 
+    5. Gently replace the SpotON sample port cover
+    6. Making sure the bung enters the SpotON port
+    7. Close the priming port and the MinION lid
```

Comment:
Will the sequencing start automatically after? What happen in the software?
We should probably add also a screenshot of the software but also way to check that everything is going well